pkg/api/v1beta2/dynakube/convert_to.go

```diff
@@ -175,38 +176,45 @@ func toAppInjectSpec(src AppInjectionSpec) *v1beta2.AppInjectionSpec {
 	return dst
 }
 
-func (src *DynaKube) toMetadataEnrichment(dst *v1beta2.DynaKube) {
+func (src *DynaKube) toMetadataEnrichment(dst *v1beta3.DynaKube) {
 	dst.Spec.MetadataEnrichment.Enabled = src.Spec.MetadataEnrichment.Enabled
 	dst.Spec.MetadataEnrichment.NamespaceSelector = src.Spec.MetadataEnrichment.NamespaceSelector
 }
 
-func (src *DynaKube) toExtensions(dst *v1beta2.DynaKube) error {
-	e := src.Spec.Extensions
+func (src *DynaKube) toExtensions(dst *v1beta3.DynaKube) error {
+	delete(dst.Annotations, api.AnnotationDynatraceExtensions)
```

Comment:
not needed

Reply:
fixed in https://github.com/Dynatrace/dynatrace-operator/commit/38d79f13166e27799292383d0c9c3571f6a58be5